scripts/kurtosis/network_params.json

```diff
@@ -2,7 +2,8 @@
     "mev_type": "full",
     "participants": [
         {
-            "el_client_type": "geth",
+            "el_client_type": "nethermind",
```

Comment:
Didn;t we got a issues with neth client?

Reply:
That was the architecture with neth peering transactions and there were a lot of them. In this architecture neth is just a validator node, broadcaster is responsible for delivering transactions to builders